src/Resources/config/services.xml

```diff
@@ -6,6 +6,7 @@
         <import resource="services/controller.xml"/>
         <import resource="services/onboarding.xml"/>
         <import resource="services/payum.xml"/>
+        <import resource="services/workflow.xml"/>
```

Comment:
Perhaps it should be loaded conditionally with `Sylius 1.13`.
```suggestion
```

Reply:
I've added conditional loader